app/client/packages/git/src/types.ts

```
@@ -0,0 +1,50 @@
+import type { PayloadAction } from "@reduxjs/toolkit";
+
+export interface GitMetadata {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)